### PR TITLE
feat(services): add uvicorn service

### PR DESCRIPTION
**Pull request by Alviner**

### CODE REVIEW

File: aiomisc/service/uvicorn.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    sock: Optional[socket.socket] = None
+
+    server: Server
+    runner: Optional[asyncio.Task] = None
+
+    def __init__(
+        self,
+        sock: Optional[socket.socket] = None,
+        config_kwargs: Optional[ConfigKwargs] = None,
```
Comment (mosquito) [inline aiomisc/service/uvicorn.py:79]: May be just `config`

Comment (mosquito) [reply aiomisc/service/uvicorn.py:79]: But really do not understans why it is not just params with defaults

File: aiomisc/service/uvicorn.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        config = Config(
+            app,
+            log_config=None,
+            use_colors=False,
+            **self.config_kwargs,
+        )
+        if not self.sock:
+            self.sock = config.bind_socket()
+        self.server = Server(config)
+        self.runner = asyncio.create_task(
```
Comment (mosquito) [inline aiomisc/service/uvicorn.py:108]:
```suggestion
        self.start_event.set()
        await self.server.serve(sockets=[self.sock])
```

File: aiomisc/service/uvicorn.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            self.server.serve(sockets=[self.sock])
+        )
+        return None
+
+    async def stop(self, exception: Optional[Exception] = None) -> Any:
+        if not self.runner:
```
Comment (mosquito) [inline aiomisc/service/uvicorn.py:114]: Do not needed because start will be canelled

Comment (Alviner) [reply aiomisc/service/uvicorn.py:114]: I'm not sure that I want just cancel start, why not call internal api and just wait to finish it the supposed way?

File: aiomisc/service/uvicorn.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            use_colors=False,
+        )
+        if not self.sock:
+            self.sock = config.bind_socket()
+        server = Server(config)
+
+        self.start_event.set()
+
+        await server.serve(sockets=[self.sock])
+        return None
```
Comment (mosquito) [inline aiomisc/service/uvicorn.py:122]: redundant return IMHO

**Review by mosquito — APPROVED** [review]

File: docs/source/locale/ru/LC_MESSAGES/api/aiomisc.po [hunk 5]
```diff
@@ -6,14 +6,14 @@ msgid ""
 msgstr ""
 "Project-Id-Version: aiomisc 16.1.16\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-07-24 13:24+0300\n"
+"POT-Creation-Date: 2023-11-21 11:47+0100\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (mosquito) [inline docs/source/locale/ru/LC_MESSAGES/api/aiomisc.po:11]: @alviner is it you?

Comment (Alviner) [reply docs/source/locale/ru/LC_MESSAGES/api/aiomisc.po:11]: no, you?)

**Pull request merged**